### ISSUE TITLE
calculateBlacklist expects integer or 'chr'+integer chromosome names

### ISSUE THREAD

**Issue by JocelynSP** [ccagc/QDNAseq#27]
Version 1.6.0 
calculateBlacklist() in 
`     pfBins1k$blacklist <- calculateBlacklist(pfBins1k, 
                                             bedFiles='CentromereTelomereRegions.bed')
`
 is giving me 
> There were 50 or more warnings (use warnings() to see the first 50)
> Warning messages:
>  1: In FUN(newX[, i], ...) : NAs introduced by coercion
>  ...
> 

All values of blacklist are either NA or 0, with no relationship to the blacklist regions.
The problem seems to be an old one that was fixed in v1.6.0 for plot() and calculateMappability() but has reappeared in the calculateBlacklist portion of createBinAnnotations. From the current Master:

```
          combined$chromosome <- sub("^chr", "", combined$chromosome)
          ....
          chr <- as.integer(x["chromosome"])
```
My Plasmodium chromosome names from PlasmoDB have format Pf3D7_[integer]_v3, plus other formats for mitochondrial and apicoplast - not surprising that it breaks

**Comment by ilarischeinin** [comment]
Did you try if it works as expected when you change that one line to:

    chr <- x["chromosome"]

**Comment by ilarischeinin** [comment]
Thanks for the bug report, and also for the code contribution to fix it.

Bioconductor 3.3 will be released on April 15th, and will include QDNAseq version 1.8.0 that will include the fix.

Meanwhile, installing QDNAseq with `devtools::install_github("ccagc/QDNAseq")` is probably the easiest way to get the fix. (Alternatively, it will also be available in QDNAseq version 1.7.5 in the development version of Bioconductor, once it makes its way there.)
